docs/technical/editor-integration.md

```diff
@@ -235,6 +235,7 @@ These editors are currently supported:
  - [JetBrains Rider](https://www.jetbrains.com/rider/)
  - [Nova](https://nova.app/)
  - [Aptana Studio](http://www.aptana.com/)
+ - [Emacs] (https://www.gnu.org/software/emacs/)
```

Comment:
@zipperer thanks for the PR. I tested it with the latest Emacs universal binary, and it works as expected 💯 
Just a small request to remove the space between the text and the link

```suggestion
 - [Emacs](https://www.gnu.org/software/emacs/)
```